docs/computing/containers/creating.md

```diff
@@ -76,3 +76,26 @@ Below is a table of common docker base images and whether installing simple pack
 |ubuntu|16.04-22.04|yes|
 
 Some issues related to, for example, glibc, fakeroot, file permissions and old remote repos are often difficult to solve, so trying out a few different base images can be a good idea before spending a lot of time debugging.
+
+## Using GPU from containers in interactive sessions in Mahti
```

Comment:
Mahti -> Puhti. www.mahti.csc.fi does not expose interactive GPUs for the desktop.